roles/website/files/Gemfile

```diff
@@ -0,0 +1,7 @@
+# Gemfile for installing Jekyll and dependencies
+
+source 'https://rubygems.org'
+gem 'jekyll', '3.0.1'
+gem 'jekyll-paginate', '1.1.0'
+gem 'redcarpet', '3.3.3'
+gem 'pygments.rb', '0.6.3'
```

Comment:
Das Gemfile gehört IMO nicht in diese Rolle sondern ins Repo der Webseite, und muss auf jeden Fall entsprechend von Jekyll ignoriert werden.